### PR TITLE
[BLOG] Add conda-grayskull-packaging

### PR DESCRIPTION
**Pull request by MarsBarLee**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents
- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

## Links to other Quansight blogs:
no labs links found
no llc links found

@noatamir 
I think I don't have permissions to add the labels Labs, Content. Could someone add those labels?

### CODE REVIEW

File: apps/labs/posts/conda-grayskull-packaging.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: 'Conda and Grayskull, the Masters of Software Packaging'
+published: January 19, 2022
+author: mahe-iram-khan
+description: 'Grayskull is an automatic conda recipe generator, with a focus on conda-forge.'
+category: [Packaging]
+featuredImage:
+  src: ../public/posts/conda-grayskull-packaging/package_name_feature.png
+  alt: 'A flow diagram of a 'package name' being fed into a gray colored skull which represents the tool Grayskull and being converted into a curled up piece of paper which represents a recipe.'
```
Comment (noatamir) [inline apps/labs/posts/conda-grayskull-packaging.md:9]:
```suggestion
  alt: 'A flow diagram of a ''package name'' being fed into a gray colored skull which represents the tool Grayskull and being converted into a curled up piece of paper which represents a recipe.'
```

Comment (noatamir) [reply apps/labs/posts/conda-grayskull-packaging.md:9]: in YAML: within single quotes the only supported escape sequence is a doubled single quote (`''`) denoting the single quote itself as in `'don''t'`

File: apps/labs/posts/conda-grayskull-packaging.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+---
+title: 'Conda and Grayskull, the Masters of Software Packaging'
+published: January 19, 2022
+author: mahe-iram-khan
+description: 'Grayskull is an automatic conda recipe generator, with a focus on conda-forge.'
+category: [Packaging]
+featuredImage:
+  src: ../public/posts/conda-grayskull-packaging/package_name_feature.png
+  alt: 'A flow diagram of a ''package name'' being fed into a gray colored skull which represents the tool Grayskull and being converted into a curled up piece of paper which represents a recipe.'
```
Comment (noatamir) [inline apps/labs/posts/conda-grayskull-packaging.md:9]:
```suggestion
  alt: "A flow diagram of a 'package name' being fed into a gray colored skull which represents the tool Grayskull and being converted into a curled up piece of paper which represents a recipe."
```

**Comment by rgommers** [comment]
@gabalafou you've approved several blog posts and added preview links (to work around the CI issue here I think?). Is there a reason these posts can't/shouldn't be merged yet? 

No particular urgency, just curious since it looks ready and I'm about to start my own first blog post with the new infra.

**Comment by gabalafou** [comment]
I reached out to @MarsBarLee on Slack and she said I could go ahead and merge these in, so I will do that now.

**Pull request merged**